the_game_V1_2.py

```diff
@@ -350,7 +350,10 @@ def move(self):

```

Comment:
when I first did this I thought why to generate the map(tiles) every time when u can save it.
Beginner me mistakes